apps/council-frontend/src/ui/proposals/GSCProposalsSection.tsx

```diff
@@ -42,8 +40,9 @@ export default function GSCProposalsSection({
   proposalsJson,
   currentBlockNumber,
 }: ProposalsSectionProps): ReactElement {
-  const { account, library } = useWeb3React();
-  const signer = useSigner(account, library);
+  const { address } = useAccount();
+  const { data } = useSigner();
+  const signer = data as Signer | undefined;
```

Comment:
Curious about this typecasting, can this be done inline safely instead?

```ts
const { data: signer } = useSigner();
```

Reply:
yeah, I thought about this too and the reason why I had to typecast it to only `Signer | undefined` is because the `wagmi` return type for `useSigner`'s `data` attribute is `FetchSignerResult | undefined` and `FetchSignerResult` extends ethers `Signer`, but also unions `null`. 

All of our function arguments/props that take in `Signer` only take into account `undefined` :(

Reply:
Ah I see, thanks for the clarification. More fallout from the wagmi refactor! We really shouldn't strip out `null` like this, though.  How much work is it to refactor our arguments/props to `Signer | null | undefined` instead?